src/java/picard/analysis/CollectWgsMetricsFromQuerySorted.java

```diff
@@ -32,20 +32,31 @@
     @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Output metrics file.")
     public File OUTPUT;
 
-    @Option(shortName = "MQ", doc = "Minimum mapping quality for a read to contribute coverage.", overridable = true)
-    public int MINIMUM_MAPPING_QUALITY = 20;
+    @Option(shortName = "USABLE_MQ", doc = "Minimum mapping quality for a read to contribute to usable coverage.", overridable = true)
+    public int MINIMUM_USABLE_MAPPING_QUALITY = 20;
 
-    @Option(shortName = "Q", doc = "Minimum base quality for a base to contribute coverage.", overridable = true)
-    public int MINIMUM_BASE_QUALITY = 20;
+    @Option(shortName = "USABLE_Q", doc = "Minimum base quality for a base to contribute to usable coverage.", overridable = true)
+    public int MINIMUM_USABLE_BASE_QUALITY = 20;
+
+    @Option(shortName = "RAW_MQ", doc = "Minimum mapping quality for a read to contribute to raw coverage.", overridable = true)
+    public int MINIMUM_RAW_MAPPING_QUALITY = 0;
+
+    @Option(shortName = "RAW_Q", doc = "Minimum base quality for a base to contribute to raw coverage.", overridable = true)
+    public int MINIMUM_RAW_BASE_QUALITY = 3;
 
     private final Log log = Log.getInstance(CollectWgsMetricsFromQuerySorted.class);
 
+    public enum METRICS_TYPE { RAW, USABLE }
+
     /** Metrics for evaluating the performance of whole genome sequencing experiments. */
     public static class QuerySortedSeqMetrics extends CollectWgsMetrics.WgsMetrics {
+        /** Identifier for metrics type */
+        public METRICS_TYPE TYPE;
```

Comment:
Oh, I was confused you're talking about the class name not the variable name.

How about FILTERING_TYPE ?